accname/name/comp_labelledby.html

```diff
@@ -13,14 +13,36 @@
 
 <p>Tests the <a href="https://w3c.github.io/accname/#comp_labelledby">#comp_labelledby</a> portions of the AccName <em>Name Computation</em> algorithm.</p>
```

Comment:
I've included several related tests for testing hidden not referenced content in ./comp_label we may be worth documenting here.

Reply:
Why here and in `./comp_label.html` rather than in `./comp_hidden_not_referenced.html` with #32

Reply:
To clarify, I'm not saying the file structure needs to stay as-is, but [I asked for feedback on the structure prior to starting](https://github.com/w3c/accname/issues/190) and didn't receive much feedback, and no conflicting opinions.

Reply:
Open to change, but for the sake of [results longevity](https://wpt.fyi/results/accname/name/comp_hidden_not_referenced.html?label=experimental&label=master&aligned&q=label%3Aaccessibility), there should be a compelling reason to rename any tests files and subtests. (click "Show history timeline" to see what I mean)

Reply:
Agreed, definitely not recommending a file structure change; I included name "precedence" tests in `./comp_label.html` so just referring to providing documentation linkages where similar stuff is tested. For example, for `comp_label.html`:

<!-- Step 2A: Hidden Not Referenced supercedes 2D: AriaLabel, also see wpt/accname/name/comp_hidden_not_referenced.html -->

Reply:
Ah got it. Thanks.

Reply:
addressed in https://github.com/web-platform-tests/wpt/pull/42522/commits/4803e83d8f62a8336aed1ff83d26cf777243b38d